Arduino/adrilight/adrilight.ino

```diff
@@ -33,118 +31,134 @@ uint8_t startIndex = 0;
 
 void setup()
 {
-    Serial.begin(1000000); // 115200
-    FastLED.clear(true);
-    FastLED.addLeds<WS2812B, LED_DATA_PIN, GRB>(leds, NUM_LEDS);
-    FastLED.setBrightness(BRIGHTNESS);
+  Serial.begin(1000000);
```

Comment:
This is not a standard bud rate. This is going screw with some Arduino boards

Reply:
I somehow managed to not see your comment until now. The baud rate is like that for some time now and at least for me this is working (arduino uno). adrilight also needs this high baud rate because it is not able to deliver 60fps with a reasonable number of LEDs.

I would love to use USB directly but this is possible, is it?